src/Hatchinq/Util.elm

```diff
@@ -59,3 +59,9 @@ keyDownAttribute keyCode message =
                 Json.Decode.fail ("not keyCode " ++ String.fromInt keyCode)
     in
     Element.htmlAttribute <| Html.Events.on "keydown" (Json.Decode.andThen isKey Html.Events.keyCode)
+
+
+{-| -}
+takeFirstTwoLines : String -> String
```

Comment:
can be refactored to `takeFirstNLines` with an additional parameter, so we can use it instead of `(\t -> String.join "\n" (List.take 1 (String.split "\n" t)))` also